### PR TITLE
Slurp returndoc

### PR DESCRIPTION
**Pull request by paultaiton**
##### SUMMARY

Add return documentation for slurp module.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

slurp
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by bmillemathias** [comment]
Hi @paultaiton 
there is conflict in your patch, could you update it please

**Comment by paultaiton** [comment]
@bmillemathias I believe I have resolved the conflict.

**Comment by bmillemathias** [comment]
> @bmillemathias I believe I have resolved the conflict.

Thanks for your contribution

**Review by bmillemathias — APPROVED** [review]

File: lib/ansible/modules/slurp.py [hunk 1]
```diff
@@ -58,7 +58,23 @@
 # 2179
 '''
 
-RETURN = r'''#'''
+RETURN = r'''
+content:
+  description: Slurpped file's content in base64.
```
Comment (acozine) [inline lib/ansible/modules/slurp.py]:
```suggestion
  description: Slurped file's content in base64.
```

File: lib/ansible/modules/slurp.py [hunk 2]
```diff
@@ -58,7 +58,23 @@
...
+  returned: always
+  type: str
+  sample: "SnVzdCBtb3JlIGNvZmZlZQo="
+source:
+  description: Path to the slurpped file.
```
Comment (acozine) [inline lib/ansible/modules/slurp.py]:
```suggestion
  description: Path to the slurped file.
```

**Review by acozine — COMMENTED** [review]
Looks good, thank you @paultaiton for adding this. Just two spelling nits - if you can accept the suggestions I'll merge the PR once it passes CI.

**Comment by paultaiton** [comment]
> Looks good, thank you @paultaiton for adding this. Just two spelling nits - if you can accept the suggestions I'll merge the PR once it passes CI.

Sure, thanks @acozine .

**Comment by samccann** [comment]
Sorry for the delay here @paultaiton - looks like in the meanwhile someone else added the return values so I'm going to close this PR but thanks again for helping out here!

**Pull request closed without merging**